### PR TITLE
Code owners

### PR DESCRIPTION
**Pull request by oded-soffrin**
Changing default reviewers according to domains
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

### CODE REVIEW

**Comment by oded-soffrin** [comment]
@lxgreen @justame @maimonav @danarish4 WDYT ?
feel free to add relevant directories to be the default PR person

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1,13 @@
-* @lxgreen @shaulgo
+# general PRs
+*   @oded-soffrin @lxgreen 
+
+# Editor Packages
+packages/tiptap-*   @justame @lxgreen @oded-soffrin
+
+# Viewer & Plugins 
+pacakges/plugin-*   @danarish4 @lxgreen @oded-soffrin
+pacakages/ricos-viewer  @danarish4 @lxgreen @oded-soffrin
```
Comment (maimonav) [inline .github/CODEOWNERS:9]:
```suggestion
packages/plugin-*   @danarish4 @lxgreen @oded-soffrin
packages/ricos-viewer  @danarish4 @lxgreen @oded-soffrin
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1 +1,13 @@
...
+
+# Platform & Schema
+packages/ricos-schema   @maimonav @lxgreen @oded-soffrin
```
Comment (maimonav) [inline .github/CODEOWNERS:12]: I would add ricos-content as well

Comment (oded-soffrin) [reply .github/CODEOWNERS:12]: si vou ple

**Review by danarish4 — APPROVED** [review]

**Pull request merged**